client/state/action-types.ts

```diff
@@ -902,14 +873,12 @@ export const SITE_IMPORTER_IS_SITE_IMPORTABLE_START = 'SITE_IMPORTER_IS_SITE_IMP
 export const SITE_IMPORTER_IS_SITE_IMPORTABLE_SUCCESS = 'SITE_IMPORTER_IS_SITE_IMPORTABLE_SUCCESS';
 export const SITE_IMPORTER_VALIDATION_ERROR_SET = 'SITE_IMPORTER_VALIDATION_ERROR_SET';
 export const SITE_KEYRINGS_DELETE = 'SITE_KEYRINGS_DELETE';
-export const SITE_KEYRINGS_DELETE_FAILURE = 'SITE_KEYRINGS_DELETE_FAILURE';
 export const SITE_KEYRINGS_DELETE_SUCCESS = 'SITE_KEYRINGS_DELETE_SUCCESS';
 export const SITE_KEYRINGS_REQUEST = 'SITE_KEYRINGS_REQUEST';
 export const SITE_KEYRINGS_REQUEST_FAILURE = 'SITE_KEYRINGS_REQUEST_FAILURE';
 export const SITE_KEYRINGS_REQUEST_SUCCESS = 'SITE_KEYRINGS_REQUEST_SUCCESS';
 export const SITE_KEYRINGS_SAVE_SUCCESS = 'SITE_KEYRINGS_SAVE_SUCCESS';
 export const SITE_KEYRINGS_UPDATE = 'SITE_KEYRINGS_UPDATE';
```

Comment:
The `SITE_KEYRINGS_UPDATE` action is also unused. We ever react only to `_UPDATE_SUCCESS`.

The same applies to `SITE_KEYRINGS_DELETE`

Reply:
> The `SITE_KEYRINGS_UPDATE` action is also unused. We ever react only to `_UPDATE_SUCCESS`.
> 
> The same applies to `SITE_KEYRINGS_DELETE`

I'll check why those aren't being flagged by the current script.